### DISCUSSION TITLE
Failed to import error when trying to implement External Scripts in frest React Router 7 install

### DISCUSSION THREAD

**Discussion by corbystephensdev** [sergiodxa/remix-utils#430] [Q&A] [answered]
I have a fairly vanilla install of RR7.0.1. I'd like to import an externally hosted JS file for a label printing function. I'm getting the following error after adding the needed bits to the root.tsx.

```
[vite] Error when evaluating SSR module /app/root.tsx: failed to import "remix-utils/external-scripts"
|- Error: Cannot find package '@remix-run/react' imported from /path-to/node_modules/remix-utils/build/react/external-scripts.js
    at packageResolve (node:internal/modules/esm/resolve:838:9)
    at moduleResolve (node:internal/modules/esm/resolve:907:18)
    at defaultResolve (node:internal/modules/esm/resolve:1037:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:650:12)
    at ModuleLoader.#cachedDefaultResolve (node:internal/modules/esm/loader:599:25)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:582:38)
    at ModuleLoader.getModuleJobForImport (node:internal/modules/esm/loader:241:38)
    at ModuleJob._link (node:internal/modules/esm/module_job:132:49)
```

This is my vite file, very "out of the box".
```
import { reactRouter } from "@react-router/dev/vite";
import autoprefixer from "autoprefixer";
import tailwindcss from "tailwindcss";
import { defineConfig } from "vite";
import tsconfigPaths from "vite-tsconfig-paths";

export default defineConfig({
  css: {
    postcss: {
      plugins: [tailwindcss, autoprefixer],
    },
  },
  plugins: [reactRouter(), tsconfigPaths()],
});
```

If I had to guess it's trying to import as if I'm still using Remix 2.x instead of React Router 7?

**Comment by sergiodxa** [comment]
That's because Remix Utils still uses Remix, the migration to RRv7 is not yet published, some utils may still work on RRv7 as they may not depend on Remix but if the dependency needs `@remix-run/react` or `@remix-run/server-runtime` it will not work with RRv7.

**Reply by corbystephensdev** [reply]
Makes sense. However, at the beginning of the Read Me file it says "This package contains simple utility functions to use with React Router." and React Router is linked to the RR7 page, not remix.run. This led me to assume that it was compatible with RR7. Maybe change that to Remix until it is RR7 compatible?

**Reply by sergiodxa** [reply, marked as answer]
The README on the repo is the latest commit, not the latest release

Also I just released v8.0.0 which now uses RRv7.
